libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -106,9 +114,22 @@ class AP_GPS_SBF : public AP_GPS_Backend
          float VPL;
     };
 
+    struct PACKED msg4014 // ReceiverStatus (v2)
+    {
+         uint32_t TOW;
+         uint16_t WNc;
+         uint8_t CPULoad;
+         uint8_t ExtError;
+         uint32_t UpTime;
+         uint32_t RxState;
+         uint32_t RxError;
+         // remaining data is AGCData, which we don't have a use for, don't extract the data
+    };
+
     union PACKED msgbuffer {
         msg4007 msg4007u;
         msg4001 msg4001u;
+        msg4014 msg4014u;
```

Comment:
48, same for me
![image](https://cloud.githubusercontent.com/assets/511568/25568725/c92bd72c-2e3b-11e7-980e-a7daee731d89.png)
so no issues